src/price/number-format.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import { SFC } from "react";
+
+export type NumberFormatProps = {
+  locale: string,
+  style: string,
+  currency: string,
```

Comment:
`currency` should be marked as optional, as it won't be passed for all number formats.

```
currency?: string,
```